neural_compressor/torch/quantization/autotune.py

```diff
@@ -54,6 +54,15 @@ def get_all_config_set() -> Union[BaseConfig, List[BaseConfig]]:
     return get_all_config_set_from_config_registry(fwk_name=FRAMEWORK_NAME)
 
 
+def _deepcopy_warp(model):
+    additional_attr_lst = ["_exported", "dynamic_shapes"]
+    original_attr = {key: getattr(model, key, None) for key in additional_attr_lst}
+    new_model = deepcopy(model)
+    for key, value in original_attr.items():
+        setattr(new_model, key, value)
```

Comment:
What will happen if we use copied attributions？

Reply:
It will raise `AttributeError`, I am not sure of the root cause.

Reply:
It seems that these attributions are not copied. It's okay.